xml/vk.xml

```diff
@@ -6213,20 +6213,20 @@ server.
         </command>
         <command successcodes="VK_SUCCESS" errorcodes="VK_ERROR_OUT_OF_HOST_MEMORY">
             <proto><type>VkResult</type> <name>vkCreateDebugUtilsMessengerEXT</name></proto>
-            <param><type>VkInstance</type> <name>instance</name></param>
+            <param externsync="true"><type>VkInstance</type> <name>instance</name></param>
```

Comment:
I think we would also need to add implicit externsync for every child of Instance. That seems unsustainable. There's also a subtle case that we probably only care about the dispatch parameters.

Maybe just add it as a freeform paragraph below `vkCreateDebugUtilsMessengerEXT`, e.g.:

> The application **must** ensure that `vkCreateDebugUtilsMessengerEXT` duration does not execute in parallel with duration of any Vulkan command that has been called with `instance` or child of `instance` as the dispatchable argument.

Reply:
I think you're statement is a little heavy handed.  "Child of `instance`" is basically any object.  I don't think it will impact anything if we just stated `instance`.

Reply:
> Child of instance" is basically any object.

Well that's intentional, no? Debug messengers affect any and all objects. E.g. command called with `device`, where that `device` is child of that `instance`.

For comparison wording of the same situation in `vkGetInstanceProcAddr`:

> The returned function pointer **must** only be called with a dispatchable object (the first parameter) that is `instance` or a child of `instance`, e.g. `VkInstance`, `VkPhysicalDevice`, `VkDevice`, `VkQueue`, or `VkCommandBuffer`.

Reply:
Ok, I don't think it's going to matter that much, but better safe than sorry.  I did clean up your wording a little, too.